src/main/java/org/craftercms/studio/impl/v2/service/content/internal/ContentServiceInternalImpl.java

```diff
@@ -27,12 +27,14 @@
 import org.craftercms.studio.api.v2.service.audit.internal.AuditServiceInternal;
 import org.craftercms.studio.model.history.ItemVersion;
 import org.eclipse.jgit.api.errors.GitAPIException;
+import org.craftercms.studio.api.v2.dal.*;
+import org.craftercms.studio.api.v2.event.content.ContentEvent;
+import org.craftercms.studio.api.v2.service.item.internal.ItemServiceInternal;
+import org.craftercms.studio.impl.v1.util.ContentUtils;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.craftercms.studio.api.v1.service.configuration.ServicesConfig;
 import org.craftercms.studio.api.v1.service.security.SecurityService;
```

Comment:
Check if this can be replaced with v2 SecurityService.
In general, let's try to replace V1 service calls as much as possible

Reply:
I added a new class `org.craftercms.studio.impl.v2.utils.ContentUtils`

Reply:
@phuongnq can we replace this `import org.craftercms.studio.api.v1.service.security.SecurityService;` with the v2 version of it?